bower.json

```diff
@@ -35,7 +36,7 @@
     "tree-model-bower": "git://github.com/BCDevOps/tree-model-bower"
   },
   "resolutions": {
-    "angular": "^1.5.9",
+    "angular": "^1.6.9",
     "lodash": "4.5.1",
```

Comment:
Lodash is listed in the package.json as v3.10.1, we should probably bump that version as well since having the resolved version being different than the declared one is weird.

Reply:
Bumped to 4.5.1 above.